articles/extensions/authorization-extension/v2/index.md

```diff
@@ -1,358 +1,39 @@
 ---
 toc: true
-description: This page explains how to setup and manage the Authorization Extension v2.
+title: Authorization Extension
+description: Guidance on setting up and managing the Authorization Extension
 ---
```

Comment:
This doc would look better with an index format, like https://auth0.com/docs/hooks